test/Sema/keypath_subscript_nolabel_mismatch.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-frontend -typecheck -verify -primary-file %s
```

Comment:
I think those tests could be unified in a single place(instead of 3 new files), in a single existing file or a new one for the rdar or SR (we some this in the [codebase](https://github.com/apple/swift/blob/master/test/Sema/rdar38885760.swift)).
 It is really just a suggestion cc @xedin

Reply:
No, I agree, that's much tidier.